tests/container.py

```diff
@@ -0,0 +1,141 @@
+            ['cp', self._container_name + ":" + container_file, host_file])
+
+    def cp_from_host(self, host_file, container_file):
+        return self.execute_docker_cmd(
+            ['cp', host_file, self._container_name + ":" + container_file])
+
+    def rm_file(self, file_name):
+        self.execute_cmd(['rm', '-rf', file_name])
+
+    def find_pid(self, process_name):
+        stdout, stderr, exit_code = self.execute_cmd(['ps', 'ax'])
+        for line in stdout.splitlines(True):
+            if "PID TTY" in line:
+                continue
+            if process_name in line:
+                return line.split()[0]
+        return -1
+
+    def get_container_name(self):
+        return self._container_name
+
+    def commit(self, name):
+        self.execute_docker_cmd(['commit', 'dune', 'dune'])
+
+    def start(self):
+        print("Starting docker container [" + self._container_name + "]")
+        self.execute_docker_cmd(['container', 'start', self._container_name])
+
+    def stop(self):
+        print("Stopping docker container [" + self._container_name + "]")
+        self.execute_docker_cmd(['container', 'stop', self._container_name])
+
+    def destroy(self):
+        print("Destroying docker container [" + self._container_name + "]")
+        self.execute_docker_cmd(['container', 'stop', self._container_name])
+        self.execute_docker_cmd(['container', 'rm', self._container_name])
+
+    def execute_cmd_at(self, directory, cmd):
+        with subprocess.Popen(['docker', 'container', 'exec', '-w', directory,
+                               self._container_name] + cmd) as proc:
+            proc.communicate()
+
+    def execute_cmd(self, cmd):
+        return self.execute_docker_cmd(
+            ['container', 'exec', self._container_name] + cmd)
+
+    def execute_interactive_cmd(self, cmd):
+        with subprocess.Popen(['docker', 'container',
+                               'exec', '-i', self._container_name] + cmd) as proc:
+            proc.communicate()
+
+    def execute_cmd2(self, cmd):
+        with subprocess.Popen(['docker', 'container',
+                               'exec', self._container_name] + cmd) as proc:
+            proc.communicate()
+
+    def execute_bg_cmd(self, cmd):
+        return self.execute_cmd(cmd + ['&'])
+
+    # possible values for the status: created, restarting, running, removing, paused, exited, dead
+    def check_status(self, status):
+        stdout, stderr, exit_code = self.execute_docker_cmd(['ps', '--filter',
+                                                             'status=' + status])
+        for line in stdout.splitlines(True):
+            if "CONTAINER ID" in line:
+                continue
+            if self._container_name in line:
+                return True
+        return False
+
+    # check if the container is still exists and was not deleted
+    def exists(self):
+        stdout, stderr, exit_code = self.execute_docker_cmd(['ps', '--filter',
```

Comment:
```suggestion
        stdout, _, _ = self.execute_docker_cmd(['ps', '--filter',
```

Reply:
Removed unused var in 1bce6e7004443ddf76013ec7ebfb142213804ca9